playbarComponent/client/src/components/App.jsx

```diff
@@ -0,0 +1,109 @@
+import React, { Component } from 'react';
+// import songFile from '../../../public/Assets/flicker.mp3';
+
+class App extends Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      currentList: null,
+      currentSong: null,
+      play: false,
+      currentTime: null,
+      duration: null,
+      volume: 0.0,
+      loop: false,
+      shuffle: false
+    };
+
+    this.handleTogglePlay = this.handleTogglePlay.bind(this);
+    this.handleNext = this.handleNext.bind(this);
+    this.handlePrevious = this.handlePrevious.bind(this);
+    this.handleShuffleToggle = this.handleShuffleToggle.bind(this);
+    this.handleRepeatToggle = this.handleRepeatToggle.bind(this);
+  }
+
+  //initialize array of songs
+  loadSongs() {
+    let songsList = [];
+    let list = this.props.songs;
+    for (let i = 0; i < list.length; i++) {
+      let song = new Audio(list[i].URL);
+      console.log('song: ', song);
+      songsList.push(song);
+    }
+    console.log('songsList: ', songsList);
+    this.setState(
+      {
+        currentList: songsList,
```

Comment:
I'm assuming that you're keeping the entire list of songs in your state for the skip buttons. Do you need to keep all of them? Or would you be able to get away with like 10?